Services/Common/OJS.Services.Common.Models/Submissions/ExecutionContext/SubmissionServiceModel.cs

```diff
@@ -71,7 +71,7 @@ public void RegisterMappings(IProfileExpression configuration)
                     opt => opt.MapFrom(s => s.SubmissionType!.ExecutionStrategyType))
                 .ForMember(
                     d => d.ExecutionType,
-                    opt => opt.MapFrom(s => ExecutionType.TestsExecution))
+                    opt => opt.MapFrom(s => Workers.Common.Models.ExecutionType.TestsExecution))
```

Comment:
Unnessessary prefix here

Reply:
It is needed. Otherwise it is causing an error in rider.

Reply:
What is the error? 
![image](https://github.com/SoftUni-Internal/OpenJudgeSystem/assets/19411274/8bbdb59c-ca93-4496-b151-73ba74baf86c)
It seems like it can be removed.

Reply:
The problem seems to be fixed. Removed the unused namespace.